src/components/bottom-bar/bottom-bar.tsx

```diff
@@ -30,7 +30,7 @@ const BottomBar: BottomBarFC = ({ children, parentRef, ...rest }) => {
   const { isScrollTop, isScrollBottom } = useVerticalScrollDirection(parentRef)
 
   return (
-    <ElBottomBar {...rest} data-slide-direction={isScrollTop ? 'up' : isScrollBottom ? 'down' : undefined}>
+    <ElBottomBar {...rest} data-is-open={isScrollTop ? 'up' : isScrollBottom ? 'down' : undefined}>
```

Comment:
**issue:** data-is-open is meant to accept `true` or `false` now, right?

**question:** I also find the use of `isScrollTop` and `isScrollBottom` a bit confusing given we just want a boolean. How simple would it be to refactor `useVerticalScrollDirection` so it's more clearly about open/closed instead of up/down?

Reply:
I think refactoring it to use context would be over-engineering at this point. The only component that needs to be aware of the open/closed state is BottomBar. It's children don't care.

Renaming the hook and having it return an `isOpen` boolean seems fine.